src/asm/rgbasm.5

```diff
@@ -812,6 +812,66 @@ first, followed by the one from
 and the one from
 .Ql bar.o
 last.
+.Ss Inline Fragments
+Inline fragments are useful for short blocks of code or data that are only referenced once.
+They are section fragments created by surrounding instructions or directives with
+.Ql [[
+double brackets
+.Ql ]] ,
+without a separate
+.Ic SECTION FRAGMENT
+declaration.
+.Pp
+The content of an inline fragment becomes a section fragment, sharing the same constraints as its parent ROM section.
+Its parent section also becomes a section fragment if it is not one already, and RGBLINK merges them together.
```

Comment:
I think you meant to say that RGBLINK will merge them as with other `SECTION FRAGMENT`s?

Reply:
RGBLINK merges the parent section and any inline fragments within it. To make that happen, the parent section itself becomes a `FRAGMENT`. Should I reword that? I didn't want to go into more detail than necessary about the implementation internals.

Reply:
I think that it's necessary, given how much it affects the end result. If there's a way to make that clear without talking about internals too much, that'd be the best of both worlds.